csharp/Platform.Ranges/Range[T].cs

```diff
@@ -19,36 +19,48 @@ public struct Range<T> : IEquatable<Range<T>>
         private static readonly EqualityComparer<T> _equalityComparer = EqualityComparer<T>.Default;
 
         /// <summary>
-        /// <para>Returns minimum value of the range.</para>
-        /// <para>Возвращает минимальное значение диапазона.</para>
+        /// <para>Gets the minimum value of the <see cref="Range{T}"/>.</para>
+        /// <para>Возвращает минимальное значение <see cref="Range{T}"/>.</para>
         /// </summary>
         public readonly T Minimum;
 
         /// <summary>
-        /// <para>Returns maximum value of the range.</para>
-        /// <para>Возвращает максимальное значение диапазона.</para>
+        /// <para>Gets the maximum value of the <see cref="Range{T}"/>.</para>
+        /// <para>Возвращает максимальное значение <see cref="Range{T}"/>.</para>
         /// </summary>
         public readonly T Maximum;
 
         /// <summary>
-        /// <para>Initializes a new instance of the Range class.</para>
-        /// <para>Инициализирует новый экземпляр класса Range.</para>
+        /// <para>Initializes a new <see cref="Range{T}"/> instance.</para>
+        /// <para>Инициализирует новый экземпляр <see cref="Range{T}"/>.</para>
         /// </summary>
-        /// <param name="minimumAndMaximum"><para>Single value for both Minimum and Maximum fields.</para><para>Одно значение для полей Minimum и Maximum.</para></param>
+        /// <param name="minimumAndMaximum">
+        /// <para>Single value for both <see cref="Minimum"/> and <see cref="Maximum"/>.</para>
```

Comment:
A single value

Reply:
For such small comments can use `suggestions` via `ctrl + g`
```suggestion
        /// <para>A single value for both <see cref="Minimum"/> and <see cref="Maximum"/>.</para>
```